flowmachine/flowmachine/core/combine_first.py

```diff
@@ -0,0 +1,98 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Collection, List, Union
+
+from flowmachine.core.query import Query
+from flowmachine.core.errors import MissingColumnsError
+
+
+class CombineFirst(Query):
+    """
+    Given two queries 'first_query' and 'other_query', fill null or missing values in the
+    result of 'first_query' using those in the result of 'other_query'.
```

Comment:
Is there a definintion of null that doesn't encompass 'not present in `first_query`, like a nodata value or similar?

Reply:
PostgreSQL 'NULL' values in first_query will be replaced by non-null values from other_query. Is that what you're asking?

Reply:
Yep, that was what I wanted to know.